algobot/helpers.py

```diff
@@ -90,65 +137,62 @@ def open_folder(folder: str):
     """
     This will open a folder even if it doesn't exist. It'll create one if it doesn't exist.
     """
-    targetPath = create_folder(folder)
-    open_file_or_folder(targetPath)
+    target_path = create_folder(folder)
```

Comment:
Nice

Reply:
Got rid of the `create_folder` function to simplify.

We could simplify the code further by creating all directories on boot - but lets too that in another PR